pkg/gofr/logging/logger.go

```diff
@@ -32,8 +32,6 @@ type Logger interface {
 	Warnf(format string, args ...interface{})
 	Error(args ...interface{})
 	Errorf(format string, args ...interface{})
-	Fatal(args ...interface{})
-	Fatalf(format string, args ...interface{})
```

Comment:
@ccoVeille - why was this removed?

Reply:
Apparently, mothing was using it, and one of them implementation was doing a deep exit:

Calling a os.Exit(1) in the code, slaughtering all the stacks and the possible defer, context cancelation.

Did I miss something and it was called?